static/js/buddy_list.js

```diff
@@ -269,7 +269,11 @@ export class BuddyList extends BuddyListConf {
 
         // Add a fudge factor.
         height += 10;
-
+        if (this.keys.length === 0) {
+            this.container = $(this.container_sel);
+            this.container.append( `<div class="required-text" data-empty="${_( 'No matching users' )}"></div>`);
```

Comment:
This is the wrong i18n interface; please read https://zulip.readthedocs.io/en/latest/translating/internationalization.html for how to do this correctly.  

Also, if we're going to use a `data-empty` system for displaying these, which we should, can we just add it to the template rather than having JS code add it?